bundles/framework/layerlist/view/LayerViewTabs/SelectedLayers/LayerBox.jsx

```diff
@@ -88,31 +122,58 @@ export const LayerBox = ({ layer, index }) => {
                                 </Row>
                                 <GrayRow>
                                     <ColAuto>
-                                        <LayerIcon style={{ marginTop: '5px' }} type={layerType} />
+                                        <LayerIcon type={layerType} />
                                     </ColAuto>
                                     <ColAuto>
-                                        <StyledSlider>
-                                            <Slider
+                                        <InputGroup compact>
+                                            <StyledSlider>
+                                                <Slider
+                                                    value={slider}
+                                                    onChange={handleOpacityChange}
+                                                    style={{ margin: '0px' }}
+                                                />
+                                            </StyledSlider>
+                                            <StyledNumberInput
+                                                min={0}
+                                                max={100}
                                                 value={slider}
                                                 onChange={handleOpacityChange}
-                                                style={{ margin: '0px' }}
+                                                formatter={value => `${value} %`}
                                             />
-                                        </StyledSlider>
+                                        </InputGroup>
                                     </ColAuto>
                                     <ColAuto>
-                                        <StyledNumberInput
-                                            min={0}
-                                            max={100}
-                                            value={slider}
-                                            onChange={handleOpacityChange}
-                                            formatter={value => `${value} %`}
-                                        />
+                                        <StyleLabel>{ locale.layer.styles.title }</StyleLabel>
```

Comment:
The style section should hidden if the layer has less than 2 styles (user can't select anything). An exception to this is layer that has the ownStyle tool (namely WFS-layers).

Reply:
Fixed the logic and moved style settings to it's own file.

Reply:
Moving this to own component makes the code a pleasure to read 👍